smallest_multiple.js

```diff
@@ -8,7 +8,22 @@
  */
 module.exports = function( ceiling ) {
   // do work here
+  let numberToBeDivided = 1;
+  let smallestNumber;
+  let noRemainders = false;
+  
+  while (!noRemainders){
+    for (i = 1; i <= ceiling; i++){
+      if (numberToBeDivided % i !== 0){
```

Comment:
Second, avoid using a negative comparison in if statements, it will increase readability and in this particular scenario bring down the lines of code.